packages/value_renderer/example/lib/main.dart

```diff
@@ -0,0 +1,175 @@
+    return Drawer(
+      child: SafeArea(
+        child: SingleChildScrollView(
+          primary: false,
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 32, vertical: 48),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                for (final item in _menu) ...[
+                  _DrawerButton(
+                    icon: item.icon,
+                    title: item.title,
+                    isSelected: item == _selectedMenuItem,
+                    onPressed: () {
+                      setState(() {
+                        _selectedMenuItem = item;
+                      });
+
+                      if (mounted) Navigator.of(context).pop();
+                    },
+                  ),
+                ],
+                const SizedBox(height: 24),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+final _menu = [
+  _MenuItem(
+    icon: Icons.description,
+    title: 'Input Examples',
+    pageBuilder: (context) {
+      return const InputExamples();
+    },
+  ),
+  _MenuItem(
+    icon: Icons.description,
+    title: 'Renderer',
+    pageBuilder: (context) {
+      return const Renderer();
+    },
+  ),
+];
+
+class _MenuItem {
+  final IconData icon;
+  final String title;
+  final Widget Function(BuildContext context) pageBuilder;
+
+  const _MenuItem({
+    required this.icon,
+    required this.title,
+    required this.pageBuilder,
+  });
+}
+
+class _DrawerButton extends StatelessWidget {
+  final IconData icon;
+  final String title;
+  final bool isSelected;
+  final VoidCallback onPressed;
+
+  const _DrawerButton({
+    Key? key,
+    required this.icon,
+    required this.title,
+    this.isSelected = false,
+    required this.onPressed,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      child: ElevatedButton(
+        style: ButtonStyle(
+            backgroundColor: MaterialStateColor.resolveWith((states) {
+              if (isSelected) {
+                return const Color(0xFFBBBBBB);
+              }
+              if (states.contains(MaterialState.hovered)) {
+                return Colors.grey.withOpacity(0.1);
+              }
+              return Colors.transparent;
+            }),
+            foregroundColor: MaterialStateColor.resolveWith((states) => isSelected ? Colors.white : const Color(0xFFBBBBBB)),
+            elevation: MaterialStateProperty.resolveWith((states) => 0),
+            padding: MaterialStateProperty.resolveWith((states) => const EdgeInsets.all(16))),
+        onPressed: isSelected ? null : onPressed,
+        child: Row(
+          children: [
+            const SizedBox(width: 8),
+            Icon(
+              icon,
+            ),
+            const SizedBox(width: 16),
+            Expanded(
+              child: Text(title),
+            ),
```

Comment:
```suggestion
            Expanded(child: Text(title)),
```
**This is a generall example**. You can leave the trailing comma out if it so the widget can be in one row if its not to long.